packages/devtools_app/lib/src/shared/status_line.dart

```diff
@@ -28,17 +30,33 @@ double get statusLineHeight => scaleByFontFactor(24.0);
 /// This displays information global to the application, as well as gives pages
 /// a mechanism to display page-specific information.
 class StatusLine extends StatelessWidget {
-  const StatusLine(this.currentScreen);
+  const StatusLine({required this.currentScreen, required this.isEmbedded});
 
   final Screen currentScreen;
+  final bool isEmbedded;
+
+  /// The padding around the footer in the DevTools UI.
+  EdgeInsets get padding => const EdgeInsets.fromLTRB(
+        defaultSpacing,
+        defaultSpacing,
+        defaultSpacing,
+        denseSpacing,
+      );
 
   @override
   Widget build(BuildContext context) {
+    final height = scaleByFontFactor(24.0) + padding.top + padding.bottom;
```

Comment:
lets use a named const instead of the raw number value

Reply:
Good point, done!